components/call_button/call_button.jsx

```diff
@@ -0,0 +1,134 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import PropTypes from 'prop-types';
+import React, {PureComponent} from 'react';
+import {injectIntl} from 'react-intl';
+
+import {intlShape} from 'utils/react_intl';
+
+import MenuWrapper from 'components/widgets/menu/menu_wrapper';
+import Menu from 'components/widgets/menu/menu';
+import CameraIcon from 'components/widgets/icons/camera_icon';
+
+const customStyles = {
+    left: 'inherit',
+    right: 0,
+    bottom: '100%',
+    top: 'auto',
+};
+
+class CallButton extends PureComponent {
+    static propTypes = {
+        currentChannel: PropTypes.object.isRequired,
+        channelMember: PropTypes.object,
+        intl: intlShape.isRequired,
+        locale: PropTypes.string.isRequired,
+        pluginCallMethods: PropTypes.arrayOf(PropTypes.object),
+    };
+
+    static defaultProps = {
+        pluginCallMethods: [],
+    };
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            menuOpen: false,
+        };
+    }
+
+    toggleMenu = (open) => {
+        this.setState({menuOpen: open});
+    }
+
+    render() {
+        const {formatMessage} = this.props.intl;
+
+        let bodyAction;
+
+        if (this.props.pluginCallMethods.length === 0) {
+            bodyAction = null;
+        } else if (this.props.pluginCallMethods.length === 1) {
+            const item = this.props.pluginCallMethods[0];
+            bodyAction = (
+                <div>
+                    <button
+                        type='button'
+                        id='calldButton'
```

Comment:
Is the id necessary? I think we've been avoiding using IDs, at least ones that are being used for styling.